echopype/convert/set_groups_ek80.py

```diff
@@ -599,7 +599,7 @@ def merge_save(ds_combine, ds_type, group_name):
                 )  # override keeps the Dataset attributes
             return set_encodings(ds_combine)
             # # Save to file
-            # io.save_file(ds_combine.chunk({'range_bin': DEFAULT_CHUNK_SIZE['range_bin'],
+            # io.save_file(ds_combine.chunk({'range_sample': DEFAULT_CHUNK_SIZE['range_sample'],
             #                                'ping_time': DEFAULT_CHUNK_SIZE['ping_time']}),
             #              path=self.output_path, mode='a', engine=self.engine,
             #              group=group_name, compression_settings=self.compression_settings)
```

Comment:
I'll remove these in https://github.com/OSOceanAcoustics/echopype/issues/506#issuecomment-1079241539